vic/vic_run/include/vic_def.h

```diff
@@ -33,6 +33,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <strings.h>
```

Comment:
I think that is fine - that's how we are doing it in VIC 5 (for example, not all the include files in vic_def.h are strictly required by all files that include vic_def.h). As long as it is needed by some of the files, then this makes sense. I just wanted to make sure it was actually needed.